src/Ocelot/Configuration/Validator/FileConfigurationFluentValidator.cs

```diff
@@ -28,7 +28,8 @@ public FileConfigurationFluentValidator(IAuthenticationSchemeProvider authentica
                 return new OkResponse<ConfigurationValidationResult>(new ConfigurationValidationResult(false));
             }
             var errors = validateResult.Errors.Select(failure => new FileValidationFailedError(failure.ErrorMessage));
-            return new ErrorResponse<ConfigurationValidationResult>(errors.Cast<Error>().ToList());
+            var result = new ConfigurationValidationResult(true, errors.Cast<Error>().ToList());
```

Comment:
I can't see any test where we check that the correct error message is being returned. Do you think we should have a test for this?